lua/autorun/sh_lastManStanding_init.lua

```diff
@@ -126,6 +128,9 @@ if SERVER then
             net.Start("ttt_lms_notify")
             net.WriteBool(true)
             net.Broadcast()
+            net.Start("lms_statistics_reveal")
+            net.WriteEntity(caller)
+            net.Broadcast()
```

Comment:
Does this need to be broadcast? Why not go with net.Send(caller) instead

Reply:
Thanks for the suggestion, I actually haven't thought of that